resources/LinuxBuild.cmake

```diff
@@ -12,21 +12,20 @@ set (DAKOTA_PYTHON ON CACHE BOOL "python interface on" FORCE)
 # DO NOT ENABLE!!! set (MPI_DEBUG OFF CACHE BOOL "mpi debug off" FORCE)
 
 ### no mpi
+set( DAKOTA_HAVE_MPI FALSE
```

Comment:
Sure I would do that too. Also should we rename this to LinuxBuildNoMPI to remove all the other cmake files which probably do not work?